parenthetics.py

```diff
@@ -0,0 +1,17 @@
+"""Parenthetics module."""
+from link_list import link_list
+
+
+def parenthetics(string):
+    """Module checks for open sets."""
+    matches = link_list()
+    for character in string:
+        if character == "(":
+            matches.append(character)
+        if character == ")":
+            if matches.head is None or matches.pop() != "(":
```

Comment:
You only ever append the opening parenthesis, so what other value would `matches.pop()` return?